feature-pack-build-maven-plugin/src/main/java/org/wildfly/build/plugin/FeaturePackBuildMojo.java

```diff
@@ -65,13 +65,13 @@
     /**
      * The configuration file used for feature pack.
      */
-    @Parameter(alias = "config-file", defaultValue = "feature-pack-build.xml", property = "wildfly.feature.pack.configFile")
+    @Parameter(alias = "config-file", defaultValue = "${basedir}/feature-pack-build.xml", property = "wildfly.feature.pack.configFile")
     private String configFile;
```

Comment:
This shouldn't change as it should only be the file name. The `configDir` property is used to determine the directory.